backend/src/main/java/ai/verta/modeldb/config/Config.java

```diff
@@ -30,6 +31,7 @@
   public TelemetryConfig telemetry;
   public TestConfig test;
   public TrialConfig trial;
+  public List<MigrationConfig> migrations;
```

Comment:
I placed this inside the database config, if I remember correctly. Could you check?

Reply:
Actually, I just saw you removed it. Sure, but I don't see any changes to the migrations file itself for some reason

Reply:
Yes you have added it in database config but it should be a part of the main config because migration is the separate thing so we had added it in root of config in old config pattern flow. that's why I have moved migration into the root config.